src/Stratis.Features.FederatedPeg.Tests/MofNMultisigTests.cs

```diff
@@ -57,10 +72,10 @@ public void Eight_Of_Fifteen_SufficientlyFunded()
                 "redeemscript=" + redeemScript.ToString(),
                 "publickey=" + keys[0].PubKey.ToHex(),
                 "federationips=0.0.0.0"
-            }));
+            }), counterChainNetworkWrapper: new Collateral.CounterChain.CounterChainNetworkWrapper(sidechainNetwork));
 
             // Construct the withdrawal tx
-            var txBuilder = new TransactionBuilder(network);
+            var txBuilder = new TransactionBuilder(sidechainNetwork);
```

Comment:
I'm not entirely sure I follow this - surely the funding transaction and withdrawal transaction need to be on the same network?

Reply:
Fixed.